packages/govern/internal/actions/DaoAction.ts

```diff
@@ -23,6 +28,20 @@ export default class DaoAction extends AbstractAction {
     super(parameters)
   }
 
+  /**
+   * Format the result to Doa
+   *
+   * @method format
+   *
+   * @returns {Promise<Dao>}
+   *
+   * @protected
+   */
+  protected format( { registryEntries }: { registryEntries: DaoEntry[] } ): Dao | null {
+
```

Comment:
Not required break

Reply:
removed this function and do the formatting in the execute() call instead.